test/test_workflow.cpp

```diff
@@ -553,6 +553,73 @@ TEST(SteadyNS_Workflow, ComponentSeparateVariable)
    return;
 }
 
+TEST(LinElast_Workflow, MFEMIndividualTest)
+{
+   config = InputParser("inputs/linelast.base.yml");
```

Comment:
You'll have to include `inputs/linelast.base.yml` in this PR

Reply:
Oops! Coming right up.